laurent22/joplin
# Mobile/Beta editor: Ctrl+F search support

# Summary
 * Adds support for Ctrl+F search and highlighting text matching the selection.
    * This can be modified (in a future PR) to support searching (perhaps from the dropdown menu).

# Testing plan
 * Open the beta editor
 * Type a short phrase
 * Type that phrase again
 * Select the first copy of that phrase

**Expected result**: All copies of the first phrase should be highlighted.
